portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,33 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.Arrays;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  // Initializes an ArrayList... Let's look more at this tomorrow.  
+  private ArrayList<String> messages = new ArrayList<String>(Arrays.asList("Hello", "This is a message.", "I am a machine..."));
```

Comment:
Can you please adjust the formatting of this line to avoid line wrap?

Maybe just add a newline before the `new` operator.